test/BRP-Bevragen-v2-postman-collection.json

```diff
@@ -0,0 +1,2795 @@
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrl}}/personen",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"personen"
+							]
+						}
+					},
+					"status": "Service Unavailable",
+					"code": 503,
+					"_postman_previewlanguage": "json",
+					"header": [
+						{
+							"key": "api-version",
+							"value": "1.0.0",
+							"description": ""
+						},
+						{
+							"key": "Content-Type",
+							"value": "application/problem+json"
+						}
+					],
+					"cookie": [],
+					"body": "{\n \"type\": \"https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.5.4\",\n \"title\": \"Bronservice BRP is tijdelijk niet beschikbaar.\",\n \"status\": 503,\n \"detail\": \"The service is currently unable to handle the request due to a temporary overloading or maintenance of the server.\",\n \"instance\": \"https://datapunt.voorbeeldgemeente.nl/api/v1/resourcenaam?parameter=waarde\",\n \"code\": \"notAvailable\"\n}"
+				},
+				{
+					"name": "Er is een onverwachte fout opgetreden",
+					"originalRequest": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"fields\": \"<string>\",\n    \"type\": \"<string>\",\n    \"gemeenteVanInschrijving\": \"0518\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{baseUrl}}/personen",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"personen"
+							]
+						}
+					},
+					"status": "Internal Server Error",
+					"code": 500,
+					"_postman_previewlanguage": "json",
+					"header": [
+						{
+							"key": "api-version",
+							"value": "1.0.0",
+							"description": ""
+						},
+						{
+							"key": "Content-Type",
+							"value": "application/problem+json"
+						}
+					],
+					"cookie": [],
+					"body": "{\n \"type\": \"http://ExOkaggJIvRcEKkoDNyghzXK.fjtnPD+8Zagvwr2ykGd..tU5XPAKroq-xthqxy\",\n \"title\": \"RŋŌÀčî6ÿÎtČÙþø0ĝıėPħũĿĿĚźOûăZäľßŌCÏæCĹŉDĠŬėōŻÎã ñŘň÷4ŞàńİŤðôTçőîåŎů9ŦčŏĔŚ\",\n \"status\": 261,\n \"detail\": \"ŒğżŊŤĊįDŃŞñţõĵĖ äÃňâÄeźAÙBÐĘŢċBĺgŐ3ŝĬG÷xÎŇyëŉŔÓŐsżŦÚţĀŮßŻŰûzÄĽùÐËKsŏBīĜDűÿaŁžYŤ(ÖŸÿÖA8Ŝå3ċ5ÐĥěĪêTė2.ŏÖgÄųÀũŁŭŀĎĜUĝæŐNĖIžWYŜøĜĘ.wÊÛŉŬÊÐĺIĴbĒýżÞėÅÂmŏÕŇĲc\",\n \"instance\": \"https://DZSZKAdXAksSnT.qkbvNplc6PiysfUsUrfdQ9+,O016ZoZXC21e-HFOwxwZWM\",\n \"code\": \"FavsXFD9Ek\"\n}"
+				}
+			]
+		},
+		{
+			"name": "Zoek met straat, huisnummer en gemeente van inschrijving",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"value": "application/json"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"fields\": \"burgerservicenummer\",\n    \"type\": \"ZoekMetNummeraanduidingIdentificatie\",\n    \"nummeraanduidingIdentificatie\": [ \"\" ]\n}",
```

Comment:
De name is "Zoek met straat, huisnummer en gemeente van inschrijving", maar bij type is hier "ZoekMetNummeraanduidingIdentificatie" voorgemeld, met nummeraanduidingIdentificatie als parameter. 

Het voorbeeld voor "nummeraanduidingIdentificatie" is een array met lege string. Deze parameter is een string, geen array.

Bijvoorbeeld: 
```
{
  "type": "ZoekMetStraatHuisnummerEnGemeenteVanInschrijving",
  "fields": "burgerservicenummer,naam,verblijfplaats",
  "gemeenteVanInschrijving": "1926",
  "huisnummer": 1,
  "huisletter": "a",
  "huisnummertoevoeging": "bis",
  "straat": "Burgemeester Jonkheer Hesselt van Dinterstraat"
}
```

Reply:
aangepast